src/_hyperscript.js

```diff
@@ -7256,7 +7256,11 @@
 
                         var smoothness = tokens.matchAnyToken("smoothly", "instantly");
 
-                        var scrollOptions = {};
+                        var scrollOptions = {
+                            block: "start",
+                            inline: "nearest"
+                        };
+
```

Comment:
Yo, can you explain this?

Reply:
> Yo, can you explain this?

https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView

Those are the defaults for scroll options according to that doc, I just explicitly set them there since I had it check the scroll option values to determine if it should offset the x / y individually.

Reply:
The thought was that if the target of a particular axis is the center of the element then it shouldn't apply the offset to that axis when positioning the temporary div. That being said, the code could probably be made clearer by writing it to check if the verticalPosition is "top" or "bottom" and / or if horizontalPosition is "left" or "right".